pkg/ovirt/logr.go

```diff
@@ -0,0 +1,68 @@
+package ovirt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	ovirtclientlog "github.com/ovirt/go-ovirt-client-log/v3"
+	"k8s.io/klog/v2/klogr"
+)
+
+type KLogr struct {
```

Comment:
Not sure if this could be moved to 
https://github.com/oVirt/go-ovirt-client-log
or https://github.com/oVirt/go-ovirt-client-log-klog
or a dedicated repo. What do you think? @janosdebugs

Reply:
Sounds good. 
Yes, I totally agree. I'd love to get a streamlined logging, but as you said - no time right now. Initially I wanted to use the `ovirtclientlog.NewGoLogger()` everywhere. However, having names such as `go-ovirt-client` or `cached-client/actuator` and being able to fine-tune log verbosity helps getting an overview of what is going on and tracking down issues. So this is just a small step.